### PR TITLE
[DAT-94] Fix url for currency endpoint

### PR DESCRIPTION
**Pull request by nreal**
# Background
Add a new URL for Doppler.currency, add prefix `conversion`

Local: http://localhost:5001/billing-currency/1/2020-09-08
Int: https://apisint.fromdoppler.net/currency/conversion/{CurrencyCode}/{date}
Qa: https://apisqa.fromdoppler.net/currency/conversion/{CurrencyCode}/{date}
Prod: https://apis.fromdoppler.com/currency/conversion/{CurrencyCode}/{date}

Swagger only in local: http://localhost:5001/swagger/index.html

### CODE REVIEW

File: Doppler.Currency/Controllers/CurrencyController.cs [hunk 1]
```diff
@@ -21,7 +20,7 @@ public class CurrencyController : ControllerBase
         public CurrencyController(ILogger<CurrencyController> logger, ICurrencyService currencyService) => 
             (_logger, _currencyService) = (logger, currencyService);
 
-        [HttpGet("{currencyCode}/{date}")]
+        [HttpGet("billing-currency/{currencyCode}/{date}")]
```
Comment (andresmoschini) [inline Doppler.Currency/Controllers/CurrencyController.cs:23]: Willthe complete URL in production be something like `https://apis.fromdoppler.com/currency/billing-currency/ARS/2020-09-08`?

File: Doppler.Currency/Startup.cs [hunk 2]
```diff
@@ -153,12 +148,16 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env, ILoggerF
                 endpoints.MapControllers();
             });
 
-            app.UseSwagger();
 
-            app.UseSwaggerUI(c =>
+            if (env.IsDevelopment())
```
Comment (andresmoschini) [inline Doppler.Currency/Startup.cs:152]: Why only in development?

File: Doppler.Currency/Startup.cs [hunk 3]
```diff
@@ -64,11 +64,6 @@ public void ConfigureServices(IServiceCollection services)
                     Description = "API for Doppler Currency"
                 });
 
-                var security = new Dictionary<string, IEnumerable<string>>
-                {
-                    { "Bearer", Array.Empty<string>() },
-                };
```
Comment (andresmoschini) [inline Doppler.Currency/Startup.cs:70]: Was it not being used?

File: Doppler.Currency/Controllers/CurrencyController.cs (hunk 1 above)
Comment (nreal) [reply Doppler.Currency/Controllers/CurrencyController.cs:23]: Yes

File: Doppler.Currency/Startup.cs (hunk 3 above)
Comment (nreal) [reply Doppler.Currency/Startup.cs:70]: Yees, This variable is not used in anything side

File: Doppler.Currency/Controllers/CurrencyController.cs (hunk 1 above)
Comment (andresmoschini) [reply Doppler.Currency/Controllers/CurrencyController.cs:23]: Mmm, and what do you think in something more representative? 

* `https://apis.fromdoppler.com/currency/conversion-value/ARS/2020-09-08?` 
* `https://apis.fromdoppler.com/currency/USD/ARS/2020-09-08?` 
* `https://apis.fromdoppler.com/currency/USD-value/ARS/2020-09-08?`

Comment (nreal) [reply Doppler.Currency/Controllers/CurrencyController.cs:23]: Yes let me check these options, I like `conversion-value` or only `conversion`, I'll change it

File: Doppler.Currency/Startup.cs (hunk 2 above)
Comment (nreal) [reply Doppler.Currency/Startup.cs:152]: Swagger needs set up correctly when a reverse proxy is used, I think that is not necessary Swagger in Prod, int, or QA, what do you think?, if It is necessary for prod, int and QA I'll create a task for it

Comment (andresmoschini) [reply Doppler.Currency/Startup.cs:152]: Oh, I understand, maybe you can add a comment in the code indicating why it is disabled.

File: Doppler.Currency/Controllers/CurrencyController.cs (hunk 1 above)
Comment (nreal) [reply Doppler.Currency/Controllers/CurrencyController.cs]: Done

File: Doppler.Currency/Startup.cs (hunk 2 above)
Comment (nreal) [reply Doppler.Currency/Startup.cs:152]: Done

**Comment by andresmoschini** [comment]
👍

**Pull request merged**